site/docs/extensions/filter-control.md

```diff
@@ -33,6 +33,19 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Default:** `false`
 
+### filterControlHidden
```

Comment:
I think `filterControlVisible` is better than `filterControlHidden`,  like our ohter options.

Reply:
OK, will change it later today.